tests/test_crc_idle.py

```diff
@@ -95,12 +95,12 @@ def test_count_idle_gpu_resources(self, mock_run_command: Mock) -> None:
 
         cluster = 'gpu'
         partition = 'default'
-        mock_run_command.return_value = "node1_4_2_idle_3500\nnode2_4_4_drain_4000"
+        mock_run_command.return_value = "node1_4_2_idle_3500\nnode2_4_4_drain_N/A"
```

Comment:
Can we make this a separate test? I.e., one test for parsing the shell output `nnode2_4_4_drain_4000` and another for `nnode2_4_4_drain_N/A`? That way we are testing both cases properly.

Reply:
Yeah, I will add that.